lib/fastlane/plugin/firebase_app_distribution/client/firebase_app_distribution_api_client.rb

```diff
@@ -84,9 +95,9 @@ def post_notes(app_id, release_id, release_notes)
       #   app_id - Firebase App ID
       #
       # Throws a user_error if the app hasn't been onboarded to App Distribution
-      def get_app(app_id, app_view = 'BASIC')
+      def get_app(app_id)
         begin
-          response = connection.get("#{v1_apps_url(app_id)}?appView=#{app_view}") do |request|
+          response = connection.get("#{v1alpha_apps_url(app_id)}?appView=BASIC") do |request|
```

Comment:
I'm confused about the need to change from `v1` to `v1alpha` here.

Reply:
`GetApp` is a v1alpha endpoint (not part of the new API).